### PR TITLE
Refactor to poetry

### PR DESCRIPTION
**Pull request by RobertLRead**
Harry, can you review this use of the Poetry package manager and merge if you have not objections?

### CODE REVIEW

**Comment by devhawk** [comment]
We should not be deleting the okf-library submodule in this PR

File: README.md [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
+# Installation and running
 
-I'm not much of a pythonista. I've installed Anaconda to get the sympy package, and I assume any user of this will have to as well.
+Installation requires four parts:
 
-You also have to instal yamale:
+1. Clone this repository with `git@github.com:helpfulengineering/project-data-platform.git --recursive`
+  - If cloned without, 
+2. Installation of Python3.8 or greater, *(left as an exercise to reader.)*
+3. Installation of the Poetry Python package-manager and build-tool.
```
Comment (devhawk) [inline README.md]: Link to Poetry install?

File: README.md [hunk 2]
```diff
@@ -1,19 +1,67 @@
...
+  - We use and recommend [Poetry](https://python-poetry.org).
+  - Alternatively, `pip` or `conda` can be used.
+4. Installation of the `okf-library` through `git submodule update`
```
Comment (devhawk) [inline README.md]: specifiying `--recursive` in step 1, do we need step 4?

File: README.md (hunk 1 above)
Comment (devhawk) [reply README.md]: never mind, I see it's farther down in the doc

File: README.md [hunk 3]
```diff
@@ -1,19 +1,67 @@
...
 
-> pip install yamale
 
-Also, we use a submodule, you may have to execute:
+## Installation and running with Poetry (recommended):
```
Comment (devhawk) [inline README.md:24]: We should have a link to the Poetry docs too

**Review by devhawk — APPROVED** [review]
THis looks good. @lynnpepin can you take a pass on the readme to add a link to the poetry install docs? the submodule stuff is also a little confusing, but if we're going to drop the submodule anyway don't spend much time on it.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,67 @@
...
 An experimental attempt to make a CLI for supply-chain modeling for Helpful Engineering's Project Data
 
-# Installation
+# Installation and running
 
-I'm not much of a pythonista. I've installed Anaconda to get the sympy package, and I assume any user of this will have to as well.
+Installation requires four parts:
 
-You also have to instal yamale:
+1. Clone this repository with `git@github.com:helpfulengineering/project-data-platform.git --recursive`
```
Comment (devhawk) [inline README.md]: should we use http: url instead of git url? `git clone https://github.com/helpfulengineering/project-data-platform.git`

**Comment by lynnpepin** [comment]
This new README with this latest push should address these! I kept the `git@` in the script with the `https://` URL included as a comment. (The `https` URL is cumbersome as a default if SSH keys are already set up. Not sure how to check this condition as a bash script.)

**Pull request merged**